login/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.contrib import admin
+from django.urls import path, include
+
+from . import views
+
+app_name = 'login'
+urlpatterns = [
+    # path('', views.index, name='index'),
```

Comment:
@zairiel243 remove this if its not needed. and possible remove all urls that are not being used